docs/app/pages/css/css-sections/icons/icon-documentation/icon-documentation.component.html

```diff
@@ -0,0 +1,67 @@
+<ux-icon
+    name="alert"
+    size="30px"
+    [rotate]="90"
+    [flipHorizontal]="false"
+    [flipVertical]="false">
+</ux-icon>
+
+<ux-icon
+    name="alert"
+    size="60px"
+    [rotate]="270"
+    [flipHorizontal]="false"
+    [flipVertical]="false">
+</ux-icon>
+
+<ux-icon
+    name="alert"
+    size="60px"
+    [flipHorizontal]="false"
+    [flipVertical]="true">
+</ux-icon>
+
+<hr>
+
+<p><code>ux-icon</code> can be used to allow a user to dynamically add icons.</p>
+
+<p>
+    The following properties can be used to configure the component:
+</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="name" type="string">
+        Defines the icon to be displayed.
+    </tr>
+    <tr uxd-api-property name="size" type="string">
+        Defines a custom size for the icon.
+    </tr>
+    <tr uxd-api-property name="rotate" type="number">
+        Defines the number of degrees to rotate the icon either <code>90</code>, <code>180</code>, <code>270</code>.
+    </tr>
+    <tr uxd-api-property name="flipHorizontal" type="boolean" defaultValue="false">
+        Define if the icon should be horizontally flipped.
+    </tr>
+    <tr uxd-api-property name="flipVertical" type="boolean" defaultValue="false">
+        Define if the icon should be horizontally flipped.
+    </tr>
+</uxd-api-properties>
+
+<h3>Overriding Custom Iconset</h3>
+
+<h4>Whole Application</h4>
```

Comment:
I'd suggest combining this with the following section, and adding a comment that `forRoot` should be replaced with `forChild` if the module is not the root module.

Reply:
Updated to include the above